test/helpers/default-functions.ts

```diff
@@ -70,52 +84,28 @@ export function defaultFunctions(): void {
 	);
 
 	/**
-	 * Get installed extensions on wiki
+	 * Skip test if extension is not installed (present) on the Wikibase server
 	 */
 	browser.addCommand(
-		'getInstalledExtensions',
-		async ( server: string ): Promise<string[] | undefined> => {
-			const result = await browser.makeRequest(
-				`${server}/w/api.php?action=query&meta=siteinfo&siprop=extensions&format=json`
+		'skipIfExtensionNotPresent',
+		async (
+			test: Context,
+			extension: string
+		): Promise<void> => {
+			const installedExtensions = await getInstalledExtensions(
+				testEnv.vars.WIKIBASE_URL
 			);
-			return lodash.map( result.data.query.extensions, 'name' );
-		}
-	);
-
-	/**
-	 * Execute docker command on container and get output
-	 */
-	browser.addCommand(
-		'dockerExecute',
```

Comment:
This command is now handled by `TestEnv#runDockerComposeCmd` as you will see applied in the `dbQuery` command above (the only place it was used). `TestEnv` is now where all Docker test service things are handled.